modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -353,3 +389,43 @@ def _get_new_qc(
             new_query_compiler._modin_frame._apply_index_objs(axis=0)
 
         return new_query_compiler
+
+    @classmethod
+    def _manage_skiprows_parameter(
+        cls,
+        skiprows: Union[int, Sequence[int], Callable, None] = None,
+        header_size: int = 0,
+    ) -> Tuple[Union[int, Sequence, Callable], bool, int]:
+        """
+        Manages read_csv skiprows parameter.
```

Comment:
It doesn't explain anything:) Please add some details.

Reply:
Corrected)